clex/driver.c

```diff
@@ -400,7 +400,13 @@ void reverse_toks (int idx)
   int matched = 0;
   int which = 0;
   int i;
+
+#ifdef _MSC_VER
+  int *saved = calloc(N, sizeof(int));
```

Comment:
as you probably noticed, this program is a quick hack anyhow. but that's a problem we can solve some other time.